charts/atlantis/values.yaml

```diff
@@ -430,6 +431,7 @@ containerSecurityContext: {}
 
 servicemonitor:
   enabled: false
+  interval: "30s"
```

Comment:
Could you add this to the readme as well?

Reply:
done